### PR TITLE
Facehuggers will no longer tear masks off if someone's head is blocked.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/tgstation/issues/25282.

### CODE REVIEW

**Comment by OtherDalfite** [comment]
Good way to begin balancing xeno. No way a facehugger should be able to tear off a full helmet.

**Comment by WJohn** [comment]
uh, thats not whats happening. whats happening is that your gas mask falls off from underneath your helmet even if the facehugger fails to actually jump onto your face.

**Comment by bgobandit** [comment]
right, because the "remove mask" code was being called before the "check helmet" code

File: code/modules/mob/living/carbon/alien/special/facehugger.dm [hunk 1]
```diff
@@ -152,22 +152,22 @@ var/const/MAX_ACTIVE_TIME = 400
...
+				return FALSE
 		if(target.wear_mask)
 			var/obj/item/clothing/W = target.wear_mask
 			if(W.flags & NODROP)
 				return FALSE
 			if(!istype(W,/obj/item/clothing/mask/facehugger))
 				target.dropItemToGround(W)
-				target.visible_message("<span class='danger'>[src] tears [W] off of [target]'s face!</span>", \
-									"<span class='userdanger'>[src] tears [W] off of [target]'s face!</span>")
+				target.visible_message("<span class='danger'>[src] tears [W] off [target]'s face!</span>", \
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: these should have ```\the``` macros on them.

File: code/modules/mob/living/carbon/alien/special/facehugger.dm [hunk 2]
```diff
@@ -152,22 +152,22 @@ var/const/MAX_ACTIVE_TIME = 400
 	// probiscis-blocker handling
 	if(iscarbon(M))
 		var/mob/living/carbon/target = M
+		if(ishuman(M))
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: monkeys can have hats now too.

Comment (bgobandit) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: is there a proc that covers humans and monkeys or do I just add an extra check

Comment (Militaires) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: isliving??? ismob??? pick

Comment (RemieRichards) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: uh... ```iscarbon``` but spoilers, you don't need the if at all anymore.

Comment (bgobandit) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: carbon includes aliens

Comment (optimumtact) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: ^

Comment (Cyberboss) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: And devils for some reason

Comment (RemieRichards) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm]: fine, ```if(ismonkey(M) || ishuman(M))```

**Comment by KorPhaeron** [comment]
XDTM finished this

**Pull request closed without merging**